lib/archethic_web/views/explorer_view.ex

```diff
@@ -199,10 +199,10 @@ defmodule ArchethicWeb.ExplorerView do
     """
   end
 
-  def format_transaction_content(:origin_shared_secrets, content) do
+  def format_transaction_content(:origin, content) do
```

Comment:
I think we have to change this as we have a single chain by family now so the content should be updated and simplified